src/app/models/pair.ts

```diff
@@ -0,0 +1,13 @@
+import {Contract} from './contract';
+import {Token} from './token';
+
+export class Pair {
```

Comment:
a better name will be Lps (LPs - liquidity pools) coz we will have different sources soon